scipy/spatial/transform/_rotation.pyx

```diff
@@ -15,12 +15,20 @@ np.import_array()
 
 # utilities for empty array initialization
 cdef inline double[:] _empty1(int n) noexcept:
+    if n == 0:
+        return array(shape=(1,), itemsize=sizeof(double), format=b"d")[:0]
```

Comment:
`shape=(0,)` not going to work?

Reply:
Using gives an error from the cython.view.array. I'm not sure if there is a more elegant way to solve that an return a placeholder for an empty array, but that's all I found.

Reply:
I've checked that you can use `np.empty` to create memory buffers, but not sure how it affects performance, so better start with `array`.

Reply:
By the way can someone explain me how the memory ownership works in this scenario? Some memory buffer is created, but then some view to it is returned. I hope it works correctly, but not super clear why.